docs/contributing.md

```diff
@@ -1,70 +1,380 @@
+
+1. Create an .env.dev file from .env.dev-sample
 
     ```bash
     cp .env.dev-sample .env.dev
     ```
 
-1. Build and run via the script
+2. Build and run the project via the script
 
     ```bash
     ./scripts/buildrun.sh
     ```
 
-1. Create a super user for logging into the web admin interface
+2. Create a super user for logging into the web admin interface
 
     ```bash
     docker-compose exec web python manage.py createsuperuser
     ```
 
-1. Browse to the web admin interface at `http://localhost:8000/admin/`
+3. Browse to the web admin interface at `http://localhost:8000/admin/` and confirm the admin site is running. Use the credentials you created in Step 2 (above) to log in.
+
+#### **1.6.a Stopping Docker**
+
+To stop the service-container, but not destroy it (often sufficient for day-to-day work):
+
+```bash
+docker-compose stop
+```
+
+To stop and destroy the service container:
+```bash
+docker-compose down
+```
+
+Add the `-v` flag to destroy the data volumes as well:
+```bash
+docker-compose down -v
+```
+
+## **Part 2: Working on issues**
+
+### **2.1 Finding an issue to work on**
+
+1. Find an issue in Prioritized Backlog [here](https://github.com/hackforla/peopledepot/projects/1)
+2. Assign the issue to yourself and move it to "In progress" column.
+3. Follow the steps in the issue description to complete the issue.
+4. Make sure to comment your ETA and Availability when you first assign yourself.
+
+### **2.2 Working on an issue**
+
+#### **2.2.a Working on an issue(1): Verifying current branch**
+Once you have selected an issue to work on, create a branch for that issue. 
+
+Verify you are on the `main` branch. 
+
+```bash
+git branch
+```
+
+You will see a list of all of your branches. There will be a star (`*`) next to the branch that you are currently in. By default you should start on the `main` branch.
+
+If you are not currently in the `main` branch, run the following command to return to it:
+
+```bash
+git checkout main
+```
+
+Create a new branch where you will work on the issue. The branch name should include the issue number. For example, to create a new branch for issue 15 and change into it:
+
+```bash
+git checkout -b <new-branch-name>-15
+```
+
+#### **2.2.b Working on an issue(2): Prepare your changes to push to your repository**
+
+Once you are done with the work on your issue you will push it to your repository.  Before you can push your work to your repository, you will stage and commit your changes.  These two commands are similar to the save command used in other programs. 
+
+**Note:** If you are using Visual studios code you can use the Git graphical user interface to stage your changes. For instructions check out the [Git Gui Wiki](https://github.com/hackforla/website/wiki/Using-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code).
+
+**Note:** Remember to run the `precommit-checks.sh` script before each commit until it can be integrated.
```

Comment:
The pre-commit is there now. It stops the commit if there's a problem, but I noticed it sometimes/never? tries to auto correct it even when it's possible. Maybe this should be a new issue to figure out. But running the script will do the auto correct. This line is still necessary until this is figured out.